aws-xray-recorder-sdk-core/src/main/java/com/amazonaws/xray/internal/SamplingStrategyOverride.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazonaws.xray.internal;
+
+public enum SamplingStrategyOverride {
+    DISABLED, // Does not override the SamplingStrategy.
```

Comment:
Not quite get why have to use DISABLED.
Can we use existing API, then update the sampled flag in SegmentImpl?

Reply:
If we are changing the parent's flag (the Lambda Frontend Segment), then will that not cause an issue in multithreading?